WooCommerce/Classes/Tools/Currency/CurrencyFormatter.swift

```diff
@@ -67,6 +67,7 @@ public class CurrencyFormatter {
         numberFormatter.generatesDecimalNumbers = true
         numberFormatter.minimumFractionDigits = decimalPosition
         numberFormatter.maximumFractionDigits = decimalPosition
+        numberFormatter.roundingMode = .halfUp
```

Comment:
Are we sure we want to follow the same behavior as the web? I'm not sure this is the right way to handle this. I've some concerns about the payments and the "bureaucracy" around it.
Eg:
If the price is `13.456$`, probably in your bank account you will receive a payment of `13.45$`. If we show `13.46$`, we are showing the wrong info to the user. Maybe it is the web that should adapt to show the correct information? 🤔

Reply:
Good point, I'll make sure to ask core why and who are they rounding up numbers.